src/data-types/connect.c

```diff
@@ -244,13 +251,15 @@ int mail_tcp_connect_with_local_address_timeout(const char * server, uint16_t po
       continue;
     
     // Christopher Lyon Anderson - prevent SigPipe
+    // patch by fdik
+if (!libetpan_deliver_sigpipe) {
```

Comment:
On Fri, Mar 06, 2015 at 07:20:51AM -0800, Hoà V. DINH wrote:
> > @@ -244,13 +251,15 @@ int mail_tcp_connect_with_local_address_timeout(const char * server, uint16_t po
> >        continue;
> >      
> >      // Christopher Lyon Anderson - prevent SigPipe
> > +    // patch by fdik
> > +if (!libetpan_deliver_sigpipe) {
> Are there situation where you'd like it not to fail?

If you mean that: yes, I'd like to get SIGPIPE.

Yours,
VB.
-- 
Volker Birk
Oberer Graben 4, 8400 Winterthur, Schweiz
mailto:vb@dingens.org  http://fdik.org